modules/sdk-coin-atom/src/atom.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  BaseCoin,
+  BitGoBase,
+  ExplanationResult,
+  KeyPair,
+  MPCAlgorithm,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  UnsignedTransaction,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
+
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+export class Atom extends BaseCoin {
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo);
+
+    if (!staticsCoin) {
+      throw new Error('missing required constructor parameter staticsCoin');
+    }
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Atom(bitgo, staticsCoin);
+  }
+
+  /**
+   * Factor between the coin's base unit and its smallest subdivison
+   */
+  public getBaseFactor(): number {
+    return 1e6;
+  }
+
+  public getChain(): string {
+    return 'atom';
+  }
+
+  public getFamily(): string {
+    return 'atom';
+  }
+
+  public getFullName(): string {
+    return 'Cosmos';
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return true;
+  }
```

Comment:
is there some account features prop we can use from static coin?

Reply:
Just commented on the other comment - but I think it might be good to add these as statics attributes in the near future. We can follow that up in a separate PR. I created a ticket - https://bitgoinc.atlassian.net/browse/BG-66189